### ISSUE TITLE
About

### ISSUE THREAD

**Issue by waynamigo** [sean-zhuh/SeqTR#25]

**Comment by waynamigo** [comment]
Thank you for sharing the code! I tried to train the model from scratch on ReferitGame dataset several times, but the result on the test set was around 67.4.
So I guess the experimental result of 69.66 in Table.2 is the result of fine-tuning on ReferitGame after pre-training on mixed datasets? or just I missed something in the paper?

Thanks,

I tested it on
- a single 2080ti GPU
- the dataset&model config files are all default from repo without any edit.

**Comment by WeitaiKang** [comment]
I have the same question as you.
In fact, i find out that the number of training sample of his annotation file is less than the normal setting of REC. For example, unc_train has 42404 samples in his annotation file, which should be 120k+ in the normal setting of REC. Same situation happens on other datasets. 
Given his performance on Tab1 and Tab2 is better than TransVG, i guess they may be trained on the combination of training set of all dataset or is trained with RIS annotation. Quite confused here....
